sympy/solvers/tests/test_solveset.py

```diff
@@ -588,7 +588,7 @@ def test_solveset_complex_polynomial():
 
 def test_sol_zero_complex():
     # This should return the complex set after it is implemented
-    raises(NotImplementedError, lambda: solveset_complex(0, x))
+    assert solveset_complex(0, x) == S.Complex
```

Comment:
``is`` will return ``True`` if two variables point to the same object, ``==`` if the objects referred to by the variables are equal. As ``S.Complex`` is a singleton, so it's the same object. However, ``==`` will also give the desired result.